docs/core/preview3/tools/dotnet-add-package.md

```diff
@@ -69,38 +69,38 @@ The package reference to add.
 
 `-h|--help`
 
-Prints out a short help for the command or argument.
+Prints out a short help for the command.
```

Comment:
again you can use `-h` for the argument `add` and/or `package` which displays different or additional information relative to that argument.